stable/wordpress/README.md

```diff
@@ -120,10 +120,11 @@ $ helm install --name my-release -f values.yaml stable/wordpress
 
 The following repo contains the recommended production settings for wordpress capture in an alternative [values file](values-production.yaml). Please read carefully the comments in the values-production.yaml file to set up your environment appropriately.
 
-To horizontally scale this chart:
+To horizontally scale this chart, first download the values-production.yaml file to your local folder, then:
```

Comment:
Would be nice if we hotlink to the file in the repo, `[values-production.yaml](values-production.yaml)`

Reply:
it is hotlink from above.. I would hot link it again.

Reply:
done

Reply:
thanks @vtuson 
@unguiculus could you please review